routes/meditation-routes.js

```diff
@@ -1,8 +1,11 @@
 const express = require('express')
+const { getSession, createVoiceSession } = require('../controllers/meditation-controller')
+
+
 const meditationRouter = express.Router()
-const { getSession } = require('../controllers/meditation-controller')
 
 /// get a meditation session
 meditationRouter.post('/sessions', getSession)
+meditationRouter.post('/createVoice', createVoiceSession)
```

Comment:
What's the difference between `POST /sessions` and `POST /createVoice`? Additionally, both `getSession` and `createVoiceSession` makes calls to OpenAI every requests. It may be beneficial to explore caching mechanism - like cdn cache. Alternatively, you can explore storing the voice information in the database to avoid re-generating?